.travis.yml

```diff
@@ -6,8 +6,6 @@ env:
     - GRADLE_OPTS="-Xmx512m"
 jdk:
   - oraclejdk8
-  - oraclejdk7
-  - openjdk7
```

Comment:
That appears to be the workaround recommended. In that case, I should close this.